Source/swiftlint/Extensions/Configuration+CommandLine.swift

```diff
@@ -48,6 +48,7 @@ private func autoreleasepool(block: () -> Void) { block() }
 
 extension Configuration {
 
+    // swiftlint:disable:next function_body_length
```

Comment:
Can we avoid making this function exceed the function body length limit? This rule is in place for a reason, it makes it hard to read and understand what the code is doing when the function is this large.

It should be possible to extract the body of the `flatMap` you added into a function, which would certainly help with readability.